rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -0,0 +1,311 @@
+# Extending task_group to manage dynamic dependencies between tasks
+
+## Introduction
+
+Back in 2021, during the move from TBB 2020 to the first release of oneTBB, 
+the lowest level tasking interface changed significantly and was no longer 
+promoted as a user-facing feature. Instead, the guidance since then has been 
+to use the task_group or the flow graph APIs to express patterns that were 
+previously expressed using with the lowest level tasking API. And for most 
+cases, this has been sufficient. However, there are some use cases that create 
+dynamically expanding graphs of dependent tasks are awkward to express using 
+the existing interfaces and so this proposal expands tbb::task_group to make 
+additional use cases easier to express.
+
+The current class definition in the latest oneTBB specification for 
+`tbb::task_group` is shown below. Note the `defer` function because this 
+function and its return type, `task_handle`, are the foundation of our proposed
+extensions:
```

Comment:
Does the defer already exist? This is not clear to me.

Reply:
Fixed